build.gradle

```diff
@@ -255,17 +241,6 @@ configure(subprojects.findAll { it.name != 'util' && it.name != 'mongo-java-driv
 
         jacoco { enabled = false }
 
-        beforeTest { descr ->
-            logger.info("[Test ${descr.className} > ${descr.name}]")
-        }
-
-        afterTest { descr, result ->
-                def failureStartTime = result.getResultType().toString() == "FAILURE" ? " Started at: ${result.getStartTime()}" : ""
```

Comment:
Why remove these?  I often rely on these log messages when searching through Evergreen build logs.

Reply:
That information is handled by the TestColorOutput.gradle class now
See: https://github.com/rozza/mongo-java-driver/pull/211/files#diff-651b2dcff7862b5ccf45b139bc5a07fbR62

Reply:
Sorry, I didn't read TestColorOutput.gradle carefully enough. 

While we're here... Can we change the output to include the string "TEST FAILURE" for failed tests instead of just "FAILURE"?  As it is now it's difficult to search the Evergreen logs in Chrome for actual test failures because so many of our test names include the word "failed" or "failure", and Chrome doesn't have case-sensitive search.

Reply:
Done.